packages/web-client/app/components/workflow-thread/index.hbs

```diff
@@ -9,7 +9,7 @@
 <Boxel::Thread
   @autoscroll={{this.autoscroll}}
   class="workflow-thread workflow-thread-animated"
-  tabindex="0"
+  tabindex="-1"
```

Comment:
@burieberry do you think this is a good change, not allowing the root to be refocused?